routes/applications.ts

```diff
@@ -24,5 +24,6 @@ router.patch(
   applicationValidator.validateApplicationUpdateData,
   applications.updateApplication
 );
+router.patch("/batch/update", authenticate, authorizeRoles([SUPERUSER]), applications.batchUpdateApplications);
```

Comment:
Here update is a verb, can we have it like `/batch` its a PATCH operation -> it updates something

Reply:
so if I do only batch, it will conflicts with /application_id route, and consider it to be a id

Reply:
understood